public/react/SaveReports/SaveReports.tsx

```diff
@@ -25,13 +25,25 @@ const SaveReport: FC<SaveReportProps> = ({ queryParamObservable, onViewReportsBu
   const openModal = () => setIsOpen(true);
   const closeModal = () => setIsOpen(false);
 
-  const reports = useAllReports();
+  const { data: reports, refetch: refetchReports } = useAllReports();
```

Comment:
had to add refetchReports to refresh the reports after adding the new one

Reply:
Why refetch instead of invalidating the query or an optimistic update? both or more inline with the rest of the app